src/components/App.js

```diff
@@ -0,0 +1,11 @@
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+const App = () => (
+  <div className="App">
+    <Display />
+    <ButtonPanel />
+  </div>
+);
+
+export default App;
```

Comment:
- [ ] THis component should have children components wrapped in React.Fragment short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) )

Please see example:

```
return (
      <React.Fragment>
        <td>Hello</td>
        <td>World</td>
      </React.Fragment>
    );
```
## Short Syntax

```
 return (
      <>
        <td>Hello</td>
        <td>World</td>
      </>
    );
```

Where the `td` element represents your child components.